src/home_robot/home_robot/agent/ovmm_agent/ppo_agent.py

```diff
@@ -224,6 +253,10 @@ def __init__(
         self.skill_start_gps = None
         self.skill_start_compass = None
 
+        embeddings_file = "clip_vit_recep_embeddings.pickle"
```

Comment:
Why is this hardcoded and not coming from a config file ?

Reply:
The sensor is initialized only in the beginning  of the skill specific agent init call,  hence hard-coded. The same was done in original ovmm sensor.py file for nav_to_obj skill.

Reply:
Okay, but in the original case the object embedings were never changed because they used CLIP embeddings only 
But now since we may experiment with different encoders, I think you need to add embeddings pickle file path as a config parameter